docs/semgrep-pro-vs-oss.md

```diff
@@ -0,0 +1,203 @@
+---
+slug: semgrep-pro-vs-oss
+append_help_link: true
+title: Semgrep Pro versus Semgrep OSS
+hide_title: true
+description: tk
+tags:
+  - Semgrep OSS
+  - Semgrep Team & Enterprise Tier
+---
+
+# Semgrep Pro versus Semgrep OSS
+
+You can use Semgrep Pro or Semgrep OSS to scan your code for security issues, bugs, and compliance to coding standards. Semgrep uses both an engine and rules to scan your code.
+
+**Rules**, which are written in YAML, describe how Semgrep generates a **finding**, such as a security issue. A rule encapsulates the pattern-matching logic and is meant to be readable and customizable.
+
+The **engine** runs an analysis using the rules you have configured it to run. Semgrep provides both a proprietary Pro engine, and an OSS engine. 
+
+This document outlines key differences between the Semgrep OSS and Pro product lines.
+
+The terms used in this document are defined as follows:
+
+<dl>
+<dt>Semgrep OSS</dt>
+<dd>Refers to Semgrep offerings with an open-source license, primarily the Semgrep OSS Engine, a fast and customizable static application security testing (SAST) scanner. To run Semgrep completely on OSS, use the OSS Engine and rules in the <a href="https://semgrep.dev/r/"><i class="fas fa-external-link fa-xs"></i> Semgrep Registry</a> with <strong>open source licenses</strong>, or write your own custom rules.</dd>
+<dt>Semgrep Pro</dt>
+<dd>Refers to proprietary product offerings from Semgrep, Inc. These include:<dl>
+<dt>Semgrep Code</dt><dd>A SAST scanner that uses cross-file (interfile) analysis for improved results over Semgrep OSS. Semgrep Code includes premium rules, known as Pro rules, that use the cross-file analysis to reduce false positives.</dd>
```

Comment:
Should we not mention interprocedural too? I see it's mentioned below, so if it's for brevity then no worries.